apps/ledger-live-mobile/src/logic/ratings.tsx

```diff
@@ -80,11 +79,9 @@ const useRatings = () => {
   const ratingsOldRoute = useSelector(ratingsCurrentRouteNameSelector);
   const ratingsHappyMoment = useSelector(ratingsHappyMomentSelector);
   const ratingsDataOfUser = useSelector(ratingsDataOfUserSelector);
-  const accounts: Account[] = useSelector(accountsSelector);
```

Comment:
This useSelector will cause a re-render of the component integrating this hook every single time the `accounts` object changes in the store.
This can be optimised as this hook only needs to know the amount of accounts with a positive balance.